### DISCUSSION TITLE
PS3 multimod, webmanmod, ps3netsrv experiences

### DISCUSSION THREAD

**Discussion by sairuk** [retronas/discussions#54] [General]
This discussion is for the PS3 service, seems the PS3 options can be a bit hit and miss with versions etc.

Issues: https://github.com/danmons/retronas/issues/48

**Comment by COIh0rp** [comment]
So I am trying to use my retronas setup for PS3 slim with CFW 4.88 Evil Nat. I have WebMod 1.47.42 and Multiman 4.85.01 both downloaded from Brewology. I have been able to synch both up with the ps3netsrv on the retronas successfully, but there are issues with both Webmod and Multiman:

Webmod:

Lists all the games in the games directory on the retronas, but commonly fails to mount the games and gives an error: Error!
or will mount the game as corrupted data.

Multiman:

Multiman will not display all the games in the games folder. It does not display  the same games as Webmod. Additionally, sometime it will display more games and sometimes less. Multiman seems to mount more successfully than Webmod.

Anyone have any ideas on how to fix these issues?

**Reply by danmons** [reply]
What's the connection for both your PS3 and your RetroNAS device?  Are they both on wired gigabit Ethernet? 

I've also come across users finding games downloaded from archive-dot-org are reported by ps3netsrv as corrupted.  They seem to run in emulators like RPCS3, but not on a real PS3 via ps3netsrv.

Are the games you're finding corrupted ISO files?  Or are they individual files in a game dir?
